packages/logs/src/rawLogsEvent.types.ts

```diff
@@ -1,29 +1,67 @@
 import type { Context, ErrorSource, TimeStamp } from '@datadog/browser-core'
 import type { StatusType } from './domain/logger'
 
-export interface RawLogsEvent {
-  date?: TimeStamp
+export type RawLogsEvent =
+  | RawConsoleLogsEvent
+  | RawNetworkLogsEvent
+  | RawLoggerLogsEvent
+  | RawAgentLogsEvent
+  | RawReportLogsEvent
+  | RawRuntimeLogsEvent
+
+export type RawLogsEventOrigin = RawLogsEvent['origin']
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is it used somewhere?